presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -264,6 +264,22 @@ public void testSimpleCoral()
                 queryAssert -> queryAssert.containsOnly(row("hive")));
     }
 
+    @Test
+    public void testNestedHiveViews()
+    {
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_base_view");
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_middle_view");
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_top_view");
+
+        onHive().executeQuery("CREATE VIEW nested_base_view AS SELECT nationkey as k, name as n, regionkey as r, comment as c FROM nation");
```

Comment:
Prepend `n_` to each column name. This is CI failure cause. 
```suggestion
        onHive().executeQuery("CREATE VIEW nested_base_view AS SELECT n_nationkey AS k, n_name AS n, n_regionkey AS r, n_comment AS c FROM nation");
```